tests/macros/tdumpast.nim

```diff
@@ -48,3 +48,9 @@ macro fun3(): untyped =
     int | float | array | seq | object | ptr | pointer | float32
   doAssert n.repr == "int | float | array | seq | object | ptr | pointer | float32", n.repr
 fun3()
+
```

Comment:
https://github.com/nim-lang/Nim/pull/13793#issuecomment-792524597 still holds

Reply:
Added to `tdumpast.nim` instead, as this is more related to the compiler than the stdlib.